### PR TITLE
Updating master branch to 3.0

### PR DESCRIPTION
**Pull request by devprincess**
This PR considers: 
- deletion of unnecessary build files 
- gitignore added
- 3.0 merged into master branch

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -117,7 +117,7 @@
     'display_github': True,
     'github_user': 'sylabs',
     'github_repo': 'singularity-admindocs',
-    'github_version': 'master',
+    'github_version': '3.0',
```
Comment (GodloveD) [inline conf.py]: Can you elaborate a bit on why this change is necessary?  What does it do?

Comment (devprincess) [reply conf.py]: This generates the link to the repo on that specific branch, it needs to be on master yes, gonna revert that change now thanks! :+1:

Comment (devprincess) [reply conf.py]: @GodloveD lmk if it's okay now :+1:
